bot/cogs/search.py

```diff
@@ -149,120 +151,187 @@ async def anime(self, ctx: Context, *, query: str) -> None:
-                            embed.add_field(name="Chapters", value=manga["attributes"]["chapterCount"])
-                        embed.add_field(name="Type", value=manga["attributes"]["mangaType"])
-                        embed.set_thumbnail(url=manga["attributes"]["posterImage"]["original"])
-
-                        try:
-                            await ctx.send(f"**{title}** - <{url}>", embed=embed)
-                        except Exception:
-                            aired = f"{manga['attributes']['startDate']}{thing}"
-                            template = textwrap.dedent(
-                                f"""
-                                ```
-                                url: {url}
-                                Title: {title}
-                                Average Rating: {manga["attributes"]["averageRating"]}
-                                Popularity Rank: {manga["attributes"]["popularityRank"]}
-                                Age Rating: {manga["attributes"]["ageRating"]}
-                                Status: {manga["attributes"]["status"]}
-                                Aired: {aired}
-                                Type: {manga['attributes']["showType"]}
-                                Powered by HotWired
-                                ```
-                                """
-                            )
-                            await ctx.send(template)
+            query = utils.escape_mentions(query)
+            query = utils.escape_markdown(query)
+
+            if not resp:
+                await ctx.send(f"No results for `{query}`.")
+                return
+
+            anime = resp[0]
+            title = f'{anime["attributes"]["canonicalTitle"]}'
+            anime_id = anime["id"]
+            url = f"https://kitsu.io/anime/{anime_id}"
+            thing = "" if not anime["attributes"]["endDate"] else f' to {anime["attributes"]["endDate"]}'
+
+            embed = Embed(title=f"{title}", color=ctx.author.color, rl=url)
+            embed.description = anime["attributes"]["synopsis"][0:425] + "..."
+            embed.add_field(name="Average Rating", value=anime["attributes"]["averageRating"])
+            embed.add_field(name="Popularity Rank", value=anime["attributes"]["popularityRank"])
+            embed.add_field(name="Age Rating", value=anime["attributes"]["ageRating"])
+            embed.add_field(name="Status", value=anime["attributes"]["status"])
+            embed.add_field(name="Aired", value=f"{anime['attributes']['startDate']}{thing}")
+            embed.add_field(name="Episodes", value=anime["attributes"]["episodeCount"])
+            embed.add_field(name="Type", value=anime["attributes"]["showType"])
+            embed.set_thumbnail(url=anime["attributes"]["posterImage"]["original"])
+            embed.set_footer(text=f"Requested by {ctx.author.name} | Powered by HotWired", icon_url=ctx.author.avatar_url_as(format="png"))
```

Comment:
I may be wrong here, but I'm pretty sure [`avatar_url()`](https://discordpy.readthedocs.io/en/latest/api.html?highlight=avatar_url#discord.User.avatar_url) should be enough for this
```suggestion
            embed.set_footer(text=f"Requested by {ctx.author.name} | Powered by HotWired", icon_url=ctx.author.avatar_url())
```